pkg/manager/manager_bgp.go

```diff
@@ -37,7 +37,7 @@ func (sm *Manager) startBGP() error {
 			log.Error(err)
 		}
 
-		// We're using Packet with BGP, popuplate the Peer information from the API
+		// We're using Packet with BGP, populate the Peer information from the API
```

Comment:
`s/Packet/Equinix Metal/`?  likewise for `s/packet/Equinix Metal/` below in the log message.

Reply:
@thebsdbox should I open a new PR to update those Packet refs?

Reply:
No, happy to keep this all under one EM change.